### PR TITLE
Added solutions for Fibonacci, Palindrome, and Smallest Even Number problems

### PR DESCRIPTION
**Pull request by FaridM5**
**This PR includes:**

**Fibonacci Solution:** Calculates the nth Fibonacci number.
**Palindrome Checker:** Determines if a given integer is a palindrome.
**Smallest Even Number:** Finds the smallest even number that is a multiple of both numbers.

### CODE REVIEW

File: src/Palindrome.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import java.util.Scanner;
+
+public class Palindrome {
+    public static boolean isPalindrome(int x) {
+        if (x >= 0) {
```
Comment (sabuhihuseynov) [inline src/Palindrome.java:5]: instead of writing else if here you could write `if(x<0) return false;`

File: src/SmallestEven.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import java.util.Scanner;
+
+public class SmallestEven {
+    public static int smallestEvenMultiple(int n) {
+        if (n % 2 == 0) {
```
Comment (sabuhihuseynov) [inline src/SmallestEven.java:5]: you can use ternar here

**Review by sabuhihuseynov — COMMENTED** [review]
pls look

File: src/Palindrome.java (hunk 1 above)
Comment (FaridM5) [reply src/Palindrome.java]: fixed✅

File: src/SmallestEven.java (hunk 2 above)
Comment (FaridM5) [reply src/SmallestEven.java]: fixed ✅

File: src/SmallestEven.java [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import java.util.Scanner;
+
+public class SmallestEven {
+    public static int smallestEvenMultiple(int n) {
+        return (n % 2 == 0) ? (n) : (n * 2);
```
Comment (sabuhihuseynov) [inline src/SmallestEven.java:5]: no need () for n and n*2

**Review by sabuhihuseynov — CHANGES_REQUESTED** [review]
pls look

**Review by sabuhihuseynov — APPROVED** [review]

**Pull request merged**